.github/workflows/test-cpp-linter.yml

```diff
@@ -1,18 +1,22 @@
-name: cpp-linter
+name: test-cpp-linter
 
 on:
   pull_request:
     types: [opened, reopened]  # let PR-synchronize events be handled by push events
   push:
   workflow_dispatch:
-
+    inputs:
+      branch_name:
+        description: 'The test branch name of cpp-linter-action'
+        required: true
+        default: 'master'
```

Comment:
This is an interesting idea! I like it.

Reply:
but it failed. I don't have much time to fix it tonight because I need to take care of the new baby :(

Reply:
Oh, congrats on the family member! No worries here. I'm a little distracted (getting the entire [nRF24 org](https://www.github.com/nRF24) setup to use our action). I'll probably have a look tomorrow.

Reply:
THANK YOU!